modules/deploy/pages/redpanda/manual/resilience/shadowing.adoc

```diff
@@ -0,0 +1,677 @@
+= Shadowing
+:description: Set up disaster recovery for Redpanda clusters using Shadowing for cross-region replication.
+:env-linux: true
+:page-categories: Management, High Availability, Disaster Recovery
+
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+
+Shadowing is Redpanda's enterprise-grade disaster recovery solution that establishes asynchronous, offset-preserving replication between two distinct Redpanda clusters. The replication stream ensures that the shadow cluster receives exact copies of the source cluster data, including offsets, timestamps, and cluster metadata. This creates a read-only shadow cluster that you can quickly promote to handle production traffic during a disaster.
```

Comment:
It works with Kafka too (which is one of the goals of the project), not sure if the wording is intentional.